docs/custom-account-fields.md

```diff
@@ -0,0 +1,182 @@
+# Account custom fields
+
+This document shows you how to setup account custom fields for your galoy instance.
+
+## Configuration
+
+```
+customFields:
+  - name: "customFieldName"
+    type: <"string" | "integer" | "float" | "boolean"> # optional (default: "string")
+    defaultValue: <value> # optional
+    editable: <true | false> # optional (default: false) - true if field can be updated by the user
+    required: <true | false> # optional (default: false) - true if field is required by mutations
+    index: <true | false> # optional (default: false) - true if field should have an index in db
+    unique: <true | false> # optional (default: false) - true if field should have a unique index in db
+```
+
+## Example
+
+Add the next configuration to `default.yaml`
+```
+accounts:
+  ...
+  customFields:
+    - name: "nationalId"
+      type: "integer"
+      defaultValue: 0
+    - name: "firstName"
+      type: "string"
+      required: true
+      editable: true
+    - name: "lastName"
+      type: "string"
+      editable: true
+    - name: "terms"
+      type: "boolean"
+      editable: true
+      defaultValue: false
```

Comment:
I think term should be more an integer. so that a user can approve any changes when terms are been updated?

0 may mean user has not accepted, but after that it's the term version number the customer has accepted.